packages/flutter_tools/lib/src/plugins.dart

```diff
@@ -50,3 +76,132 @@ void writeFlutterPluginsList() {
     }
   }
 }
+
+const String _androidPluginRegistryTemplate = '''package io.flutter.plugins;
+
+import io.flutter.app.FlutterActivity;
+
+{{#plugins}}
+import {{package}}.{{class}};
+{{/plugins}}
+
+/**
+ * This file is generated. Do not edit.
+ */
+
+public class PluginRegistry {
+{{#plugins}}
+    {{class}} {{name}};
```

Comment:
Added public visibility.

I'm not too worried about users modifying the fields, since the PluginRegistry is only used by the app. If it was used by the framework, I'd want to lock it down.

We should revisit the plugin "interface" when/if we move initialization to FlutterApplication, since that'll heavily influence how they're registered.